utils/rpms/daos.spec

```diff
@@ -168,18 +170,23 @@ Requires: mercury >= %{mercury_version}
 Requires: libfabric >= %{libfabric_version}, libfabric < %{libfabric_max_version}
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
+Requires: capstone >= 4.0.1
```

Comment:
Is it really required to explicitly require this?  That is, are there binaries, tools, etc. (in fact I only see /usr/bin/cstool in this package) that are required from this package or is it just the shared library that is needed?  If the latter, this requirement will be added to the RPM automatically and does not need to be expressed explicitly.

I don't see `cstool` appearing anywhere in this patch so probably it is not being used, yes?  If so, then this should be removed here.

```suggestion
```

Reply:
only the shared library is needed. I think this is not required. It was suggested to add here to avoid the issue of missing libcapstone.so in unit test in CI. I will remove it then. Thank you!

Reply:
I have no idea about cstool.